server/repository/src/tests.rs

```diff
@@ -246,24 +246,6 @@ mod repository_test {
                 email: None,
             }
         }
-
-        pub fn central_sync_buffer_row_a() -> CentralSyncBufferRow {
```

Comment:
I really like the idea of putting tests closer to logic being tested, I think tests in separate files only when logic being tested spans between multiple domains/modules or when there is dependencies from multiple modules or domains

Reply:
This was relocated to sync_buffer repo

Reply:
agree, if I remember correctly there was the option to have SqliteRepos and PGRepos and the tests where suppose to run on both repo implementations and with the way the project was structured they all ended up in one file...

Reply:
cool, now I remember the reason for it.